protocol/cmd/dydxprotocold/cmd/config.go

```diff
@@ -12,11 +12,11 @@ const (
 	// `minGasPriceUusdc` is default minimum gas price in micro USDC.
 	minGasPriceUusdc = "0.025" + assettypes.UusdcDenom
 	// `minGasPriceStakeToken` is the default minimum gas price in stake token.
-	// TODO(GENESIS): `dv4tnt` is a placeholder for the stake token of the dYdX chain.
-	// Before this software is published for genesis, `dv4tnt` should be replaced with
+	// TODO(GENESIS): `adv4tnt` is a placeholder for the stake token of the dYdX chain.
+	// Before this software is published for genesis, `adv4tnt` should be replaced with
 	// the chain stake token. It's also recommended that the min gas price in stake token
 	// is roughly the same in value as 0.025 micro USDC.
-	minGasPriceStakeToken = "0.025dv4tnt"
+	minGasPriceStakeToken = "0.025adv4tnt"
```

Comment:
does this have to change value by 12 orders of magnitude given the comment above this line?

Reply:
Planned to do this in a future PR, but might as well do it here.